pkg/meta/meta.go

```diff
@@ -0,0 +1,55 @@
+package meta
+
+import "github.com/k8ssandra/k8ssandra-operator/pkg/utils"
+
+// +kubebuilder:object:generate=true
+type MetaTags struct {
+	// +optional
+	Labels map[string]string `json:"labels,omitempty"`
+
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
+}
+
+// merge MetaTags into a destination MetaTags. If properties are shared between the two configs,
+// the destination MetaTags property will be used
+func MergeMetaTags(m1 *MetaTags, m2 *MetaTags) *MetaTags {
```

Comment:
This function should go away when we switch to using Goalesce to merge the meta tags.